src/main/java/com/rakovets/course/java/core/practice/lambda_expressions/city_helper/CityHelper.java

```diff
@@ -0,0 +1,35 @@
+package com.rakovets.course.java.core.practice.lambda_expressions.city_helper;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class CityHelper {
+    public static List<String> getListCitiesWithUniqueName(List<String> city) {
+        List<String> uniqueCityNames = new ArrayList<>();
+        city.stream().distinct().forEach(x -> uniqueCityNames.add(x));
```

Comment:
This is not readable
```suggestion
        city.stream()
                .distinct()
                .forEach(x -> uniqueCityNames.add(x));
```